docs/adapters/http.md

```diff
@@ -34,6 +38,10 @@ Parameter         |             Description          | Required?
 `-includeHeaders` | When set to true the headers from the response are appended to each of the points in the response | No; default: `false`
 `-timeField`      | Name of the field in the data which contains a valid timestamp  | No; defaults to `time`
 `-format`         | HTTP data format, supports: `csv`, `json` and `jsonl` | No; defaults to HTTP response header `Content-Type`
+`-followLinkHeader` |  When set to true the [Web Linking](https://tools.ietf.org/html/rfc5988) feature is enabled which allows for automatic handling of paging as specified by the [RFC5988](https://tools.ietf.org/html/rfc5988). | No; default `false`
+`-pageParam`      | The name of the query parameter used to enable paging with multiple HTTP requests sent to the URL provided until response contains 0 records | No
```

Comment:
@dmehra the default is its not set and not required since it would enable a "behavior".